images/inference-services/kfserving/run.sh

```diff
@@ -34,19 +41,23 @@ if [ "${PREDICTOR}" = "auto" ]; then
     fi
 fi
 
-export isvc="${ORG}-${PROJECT}-${FUSEML_ENV_WORKFLOW_NAME}"
 case $PREDICTOR in
     # kfserving expects the tensorflow model to be under a numbered directory,
-    # however mlflow saves the model under 'tfmodel', so if there is no directory
+    # however mlflow saves the model under 'tfmodel' or 'data/model', so if there is no directory
     # named '1', create it and copy the tensorflow model to it.
-    tensorflow)
-        if [ "$(mc find ${model_bucket} --name 1)" = "" ]; then
-            mc cp -r "${model_bucket}"/tfmodel/ "${model_bucket}"/1
+    keras|tensorflow)
```

Comment:
Why add `keras` as a predictor? AFAIK there is no `keras` kind of predictor that is supported by kfserving.

Reply:
You're right, there isn't. But MLFlow does encode this differently in the generated `MLmodel` file when it comes from Keras, and we need to deal with it here. This is only useful when the predictor parameter is set to `auto` and the model is a Keras/MLFlow model. A few lines below, I change it back to `tensorflow`.

Reply:
But if the predictor parameter is set to `auto` `$PREDICTOR` will never be `keras`, see: https://github.com/fuseml/extensions/blob/58194eb24ceb30ea7a16173443c789de7caf67aa/images/inference-services/kfserving/run.sh#L36

Reply:
Ok, so based on our conversation, using `keras` as a predictor value sends the wrong idea. I'll think of a different way to fix this. For now, I'm removing the keras changes from this PR.